### ISSUE TITLE
Suggestion: Flat / Touchscreen Buttons and Switches

### ISSUE THREAD

**Issue by MatCat** [tsunamayo/Starship-EVO#3049]
A simple flat brick that basically is a touch screen switch and button (one for each with their own unique graphic), a lot of sci-fi ships don't use big clunky 3D switches and buttons, often they have touch screen controls, just gives another aesthetic option, also being flat like a decal would be non-blocking to bricks and blocks in its space.

**Comment by tsunamayo** [comment]
Hi, do you mean something like the flat button but decal form? Because we already have a flat button (in a wedge form). thx

**Comment by MatCat** [comment]
I mean flat like a decal, so it doesn't have any 3D shape at all, no handles, just looks like a little screen with a digital switch or button

**Comment by CurioInventorium** [comment]
Decal switches and logic stuff would definitely be a nice addition.
It's like we're at Alien tech-level of switches and buttons, with Decal style it would allow TNG interfaces.
All we'd need then is Steampunk style switches and dials :D

**Comment by StarrKiss** [comment]
I agree- lets have ALL the levels of tech lol. There are so many cool possibilites to be explored with all 3 styles!

**Comment by WILDCATreactor** [comment]
@StarrKiss agree but disagree, what we need is one of each block-type of asset, workshop mods, if that feature ever opens up after beta/release, will allow us to add any number of themes to those blocks, all Tsunamayo really needs to do is provide coding functionality and examples so those add-ons to the base game can be made.  I'm perfectly willing to wait forever on those things so he can focus on the game's core systems.
